src/components/CoursePage/InvestInYourFuture.js

```diff
@@ -1,4 +1,4 @@
-const InvestInYourFuture = () => {
+const InvestInYourFuture = ({ investInYourselfBtnURL }) => {
```

Comment:
This URL in unused

Reply:
its unused for now but once we add the form to add enroll, we can just add the link there, I added it there so we don't forget to add it

Reply:
if both the buttons link to the same course form, cant we just pass one prop through both components instead of 2 separate?

Reply:
If you mean all courses lead to the same enroll form, then we can just hard code the link